packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -311,6 +311,7 @@ function UntranslatedComplexItem(props) {
 				{info.keys.map((key, i) => {
 					const childId = id && `${id}-${key}`;
 					return (
+						// eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
it's a circular dependency between the Item and ComplexItem,so disable the eslint check.

Reply:
:sweat_smile: